docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/000_nav_blocks.content.yml

```diff
@@ -7,11 +7,11 @@
   status: 1
```

Comment:
This content should have been deleted. I wonder how it is appearing here...
Yeah, it was removed in https://github.com/NCIOCPL/cgov-digital-platform/pull/1646.
And somehow it was added back in as part of https://github.com/NCIOCPL/cgov-digital-platform/commit/5ab0625c373eb98b629f516956b5ea939613bfb1#diff-509092f44cecf9a817d95f13e6ddae83

So, please kill this file.